src/mongoose_internal_databases.erl

```diff
@@ -29,10 +29,11 @@ init(mnesia, #{}) ->
         _ ->
             ok
     end,
-    application:start(mnesia, permanent),
+    application:ensure_all_started([mnesia], permanent),
     wait_for_mnesia(),
     mongoose_node_num_mnesia:init();
 init(cets, #{}) ->
+    application:ensure_all_started([cets], permanent),
```

Comment:
AFAIR, @arcusfelis was reluctant to do this when I suggested it some time ago... I don't remember why. Let's hope CETS doesn't break in some strange way now.

Reply:
To be very precise, absolutely nothing will happen, this is just for consistency. https://github.com/esl/cets/blob/main/src/cets.app.src has no declared modules to start other than depending on `kernel` and `stdlib` just like every other app and library.